dev_build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2018 David Hopper(davidhopper2003@hotmail.com). All Rights Reserved.
```

Comment:
Might want to keep the author info aligned with others as 'Apollo Authors'.

Reply:
OK. No problem.